build.gradle.kts

```diff
@@ -1,3 +1,9 @@
+buildscript {
+	dependencies {
+		classpath("com.google.android.gms:oss-licenses-plugin:0.10.4")
```

Comment:
Used 10.4 version to see the licences list :D (10.6 is not showing the list when the app is not in release version)